modules/nw-operator-vsphere-multiple-nic.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-operator-vsphere-multiple-nic.adoc_{context}"]
+= Specifying multiple network interface cards (NICs)
+
+Before you install an {product-title} cluster on a {vmw-short} host, you can specify multiple NICs for a networking implementation so that the {vmw-short} cloud controller manager (CCM) can select the appropriate NIC for a given networking situation. The {vmw-full} can use the NIC for managing pods and services on your cluster. For this configuration, you must specify internal and external Classless Inter-Domain Routing (CIDR) implementations in the {vmw-short} cloud controller manager (CCM) configuration. Each CIDR lists an IP address range that the CCM uses to determine what NICs interact with traffic from internal and external networks.
+
+If you do not configure internal and external CIDRs in the {vmw-short} CCM configuration, the {vmw-short} CCM selects the first NIC that the host operating system reports as the primary network interface. This configuration can cause new nodes that associate with a `MachineSet` object with a single NIC to become unusable as each new node receives the `node.cloudprovider.kubernetes.io/uninitialized` taint. In certain situations, the reported network interface might not be the expected network interface. These situations can cause communication issues with the Kubernetes API server that can cause installation of the cluster to fail.
+
+[NOTE]
+====
+If you configured multiple NICs for a network implementation but only intend to use a single NIC for your current needs, ensure that you specify the same IP address for the internal and external CIDRs. This configuration ensures that the CCM selects your preferred NIC and not the first NIC that the host operating system reports.
+====
+
+.Example of an error message outputted because the {vmw-short} CCM selected an incompatible NIC
+[source, text]
+----
+ERROR Bootstrap failed to complete: timed out waiting for the condition
+ERROR Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane.
+----
+
+.Prerequisites
+
+* You created Kubernetes manifest files for your {product-title} cluster.
+
+.Procedure
+
+. From the directory where you store your {product-title} cluster manifest files, use your preferred text editor to open the `manifests/cluster-infrastructure-02-config.yml` manifest file.
+
+. Add an `nodeNetworking` object to the file and specify internal and external network subnet CIDRs for the object.
```

Comment:
I think this article should be "a" since the next word you pronounce is "node":
```suggestion
. Add a `nodeNetworking` object to the file and specify internal and external network subnet CIDRs for the object.
```